inventory/hosts

```diff
@@ -169,6 +183,7 @@ us-prod
 europe
 america
 au-prod
+br-prod
```

Comment:
this group is for servers maintained by the global team, so BR, for now, should not be here.
you can delete this line.

Reply:
This line needs to be removed :fire: